src/open_inwoner/components/templatetags/header_tags.py

```diff
@@ -19,3 +19,30 @@ def accessibility_header(request, **kwargs):
     config = SiteConfiguration.get_solo()
     kwargs["help_text"] = config.get_help_text(request)
     return {**kwargs, "request": request}
+
+
+@register.simple_tag(takes_context=True)
+def display_search(context):
```

Comment:
Let's underline in the naming it can differ per user.

```suggestion
def display_search_for_user(context):
```